index.js

```diff
@@ -56,7 +56,9 @@ const create = (win, options) => {
 				label: 'Search with Google',
 				visible: hasText,
 				click() {
-					const url = new URL('https://www.google.com/search?q=' + props.selectionText).href;
+					const queryParams = {q: props.selectionText};
+					const queryString = new URLSearchParams(queryParams).toString();
+					const url = new URL('https://www.google.com/search?' + queryString).href;
```

Comment:
This is not what I asked for. I asked you to use `URL#searchParams`.  `const url = new URL(…); url.searchParams.set(…);`.

Reply:
And use `url.toString()`.

Reply:
Did this in the following commit, but am curious, why can't we use `const url = new URL('https://www.google.com/search?' + new URLSearchParams(queryParams).toString()).href;`

Is it just because of readability or it due to some other reason as well?

Reply:
You technically could, but what if you forget to put the `?` at the end. I prefer using dedicated APIs for formatting whenever possible to prevent mistakes.